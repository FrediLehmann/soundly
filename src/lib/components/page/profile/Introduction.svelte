<h1>Profile</h1>

<style>
  h1 {
    font-size: 1.75rem;
    padding-block-end: 0.5rem;
    margin-block-end: 2rem;
    border-bottom: 1px solid hsla(var(--neutral-200));
  }
</style>
