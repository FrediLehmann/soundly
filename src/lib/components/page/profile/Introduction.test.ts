/**
 * @jest-environment jsdom
 */

import '@testing-library/jest-dom';
import { render } from '@testing-library/svelte';

import Introduction from './Introduction.svelte';

describe('<Introduction /> component', () => {
  test('renders correctly', () => {
    const { component } = render(Introduction);
    expect(component).toMatchSnapshot();
  });
});
