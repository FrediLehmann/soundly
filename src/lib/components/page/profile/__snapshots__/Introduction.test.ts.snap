// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Introduction /> component renders correctly 1`] = `
Introduction {
  "$$": Object {
    "after_update": Array [],
    "before_update": Array [],
    "bound": Object {},
    "callbacks": Object {},
    "context": Map {},
    "ctx": Array [],
    "dirty": Array [
      -1,
    ],
    "fragment": Object {
      "c": [Function],
      "d": [Function],
      "i": [Function],
      "l": [Function],
      "m": [Function],
      "o": [Function],
      "p": [Function],
    },
    "not_equal": [Function],
    "on_destroy": Array [
      [Function],
    ],
    "on_disconnect": Array [],
    "on_mount": Array [],
    "props": Object {},
    "root": <div>
      <h1
        class="svelte-dwpor9"
      >
        Profile
      </h1>
    </div>,
    "skip_bound": false,
    "update": [Function],
  },
}
`;
